scripts/upgrade

```diff
@@ -123,7 +123,7 @@ ynh_script_progression --message="Setting up sandobx domain: $sandboxdomain" --w
 # We don't test that in CI
 if ! [ ${PACKAGE_CHECK_EXEC:-0} -eq 1 ]; then
 	# If sandboxdomain doesn't exist, create it
-	if [ -n "$sandboxdomain" ]; then
+	if yunohost domain list --output-as json | jq .domains[] | tr -d '"' | grep -q "$sandboxdomain" != "$sandboxdomain"; then
```

Comment:
```suggestion
	if ! yunohost domain list --output-as json | jq -r .domains[] | grep -q "^$sandboxdomain\$"; then
```

- `-r` can be used to have a jq output "without quotes"
- grep should have `^` / `$` to be strict, just to cover some boring edge case where the sandbox domain could be `foo.bar.tld` but `foo.bar.tld.zblerg` would exist somehow ...
- writing `if cmd == 'foobar'` doesnt make sense ... you either want `if cmd` (meaning you want to check that a command succeeds) or `if [ "$(cmd)" == "foobar" ]` (meaning that the stdout of a command if foobar). In the current context, `grep -q` will return 0 or 1 depending on if it found a match of not